tsl/src/compression/compression.c

```diff
@@ -1443,6 +1443,7 @@ decompress_chunk(Oid in_table, Oid out_table)
 	FreeBulkInsertState(decompressor.bistate);
 	MemoryContextDelete(decompressor.per_compressed_row_ctx);
 	ts_catalog_close_indexes(decompressor.indexstate);
+	truncate_relation(in_table);
```

Comment:
Its not a bug, previously decompressing a chunk would drop it at a later point. Now we truncate it instead of dropping. I could move this to the place where we used to drop it just for clarity.

Reply:
No reason to optimize the code for the reviewer, you can keep it where it is. It just looked like it was a bug.